exchange/non_bid_reason.go

```diff
@@ -22,3 +25,11 @@ func (n *NonBidReason) Val() NonBidReason {
 	}
 	return *n
 }
+
+func ErrorToNonBidReason(errorCode int) NonBidReason {
+	switch errorCode {
+	case errortypes.TimeoutErrorCode:
+		return ErrorTimeout
```

Comment:
And is there a way to test this function so that this line is covered?

Reply:
Covered via https://github.com/prebid/prebid-server/blob/d4754a15637871e4585f44454e75e161e5c9479b/exchange/bidder_test.go#L3060